processor/src/main/java/org/mapstruct/ap/internal/model/AbstractBaseBuilder.java

```diff
@@ -39,6 +41,11 @@ public B method(Method sourceMethod) {
         return myself;
     }
 
+    public B versionInformation(VersionInformation versionInformation) {
```

Comment:
I would actually add the `VersionInformation` to the `MappingBuilderContext`, no need to pass it along the builder.